src/games/gamesContainer.py

```diff
@@ -88,7 +88,7 @@ def findGameByUuid(self, uuid):
 
     def addGame(self, game):
         """Add a game to the list"""
-        if not game in self.games :
+        if game not in self.games :
```

Comment:
At that rate we'll still have them by the time protons start decaying. What's the point of fixing it here if there are another 100 of these in the file? Better to fix them all in one go IMO.